jsonschema-core.md

```diff
@@ -895,49 +895,44 @@ by other parties.
 ### Base IRI, Anchors, and Dereferencing
 
 To differentiate between schemas in a vast ecosystem, schemas are identified by
-[IRI](#rfc3987), and can embed references to other schemas by specifying their
-IRI.
+[absolute IRIs](#rfc3987) (without fragments) and can embed references to other
+schemas by specifying their respective IRIs. When comparing IRIs for the
+purposes of resource identification, implementations SHOULD first follow the IRI
+normalization procedures defined in [RFC 3987](#rfc3987), section 5.3.
 
-Several keywords can accept a relative [IRI-reference](#rfc3987), or a value
-used to construct a relative IRI-reference. For these keywords, it is necessary
+Several keywords can accept a relative [IRI reference](#rfc3987), or a value
+used to construct a relative IRI reference. For these keywords, it is necessary
 to establish a base IRI in order to resolve the reference.
 
 #### The `$id` Keyword {#id-keyword}
 
-The `$id` keyword identifies a schema resource with its [canonical](#rfc6596)
-IRI.
+The `$id` keyword identifies a schema resource. The value for this keyword MUST
+be a string, and MUST represent a valid [IRI reference](#rfc3987) (without a
+fragment).
```

Comment:
I suggest removing the parens in this statement. Previously, we used this with "absolute-IRI". In that case, the definition of absolute IRI already disallows a fragments and the parenthetical is just a helpful reminder of that fact.

However, fragments _are_ normally allowed in IRI references, so disallowing fragments is an important part of the requirement, not an aside, which makes it confusing why it's in parens.

```suggestion
The `$id` keyword identifies a schema resource. The value for this keyword MUST
be a string, and MUST represent a valid [IRI reference](#rfc3987) without a
fragment.
```

Reply:
Was thinking similarly.